lang/lang_en_hr.txt

```diff
@@ -1,950 +1,950 @@
 
 #MSG_WIZARD_CLEAN_HEATBED c=20 r=8
 "Please clean heatbed and then press the knob."
-"\x00"
+"Ocistite grijacu podlogu, a zatim pritisnite gumb."
 
 #MSG_CONFIRM_NOZZLE_CLEAN c=20 r=8
 "Please clean the nozzle for calibration. Click when done."
-"\x00"
+"Molimo ocistite mlaznicu radi kalibracije. Kliknite kada ste gotovi."
 
 #MSG_SELFTEST_PLEASECHECK c=20
 "Please check:"
-"\x00"
+"Molimo provjerite:"
 
 #MSG_WIZARD_CALIBRATION_FAILED c=20 r=8
 "Please check our handbook and fix the problem. Then resume the Wizard by rebooting the printer."
-"\x00"
+"Provjerite nas prirucnik i rijesite problem. Zatim nastavite s Carobnjakom ponovnim pokretanjem printera."
 
 #MSG_CHECK_IDLER c=20 r=5
 "Please open idler and remove filament manually."
-"\x00"
+"Molimo otvorite klizac i rucno uklonite filament."
 
 #MSG_PLACE_STEEL_SHEET c=20 r=5
 "Please place steel sheet on heatbed."
-"\x00"
+"Stavite celicnu plocu na grijacu podlogu."
 
 #MSG_PRESS_TO_UNLOAD c=20 r=4
 "Please press the knob to unload filament"
-"\x00"
+"Pritisnite gumb za praznjenje filamenta"
 
 #MSG_PULL_OUT_FILAMENT c=20 r=4
 "Please pull out filament immediately"
-"\x00"
+"Molimo odmah izvucite filament"
 
 #MSG_EJECT_REMOVE c=20 r=4
 "Please remove filament and then press the knob."
-"\x00"
+"Molimo uklonite filament i zatim pritisnite gumb."
 
 #MSG_REMOVE_STEEL_SHEET c=20 r=4
 "Please remove steel sheet from heatbed."
-"\x00"
+"Molimo uklonite celicnu plocu sa grijace podloge."
 
 #MSG_RUN_XYZ c=20 r=4
 "Please run XYZ calibration first."
-"\x00"
+"Prvo pokrenite XYZ kalibraciju."
 
 #MSG_UPDATE_MMU2_FW c=20 r=4
 "Please update firmware in your MMU2. Waiting for reset."
-"\x00"
+"Azurirajte firmwer u svom MMU2. Ceka se resetiranje."
 
 #MSG_PLEASE_WAIT c=20
 "Please wait"
-"\x00"
+"Molimo pricekajte"
 
 #MSG_REMOVE_SHIPPING_HELPERS c=20 r=3
 "Please remove shipping helpers first."
-"\x00"
+"Najprije uklonite prijevozne osloce."
 
 #MSG_PREHEAT_NOZZLE c=20
 "Preheat the nozzle!"
-"\x00"
+"Predgrij. mlaznicu!"
 
 #MSG_PREHEAT c=18
 "Preheat"
-"\x00"
+"Predgrijavanje"
 
 #MSG_WIZARD_HEATING c=20 r=3
 "Preheating nozzle. Please wait."
-"\x00"
+"Predgrijavanje mlaznice. Molim vas pricekajte."
 
 #MSG_NEW_FIRMWARE_PLEASE_UPGRADE c=20
 "Please upgrade."
-"\x00"
+"Molimo nadogradite."
 
 #MSG_PRESS_TO_PREHEAT c=20 r=4
 "Press the knob to preheat nozzle and continue."
-"\x00"
+"Pritisnite gumb za predgrijavanje mlaznice i nastavite."
 
 #MSG_FS_PAUSE c=5
 "Pause"
-"\x00"
+"Pauza"
 
 #MSG_POWER_FAILURES c=15
 "Power failures"
-"\x00"
+"Prekidi struje"
 
 #MSG_PRINT_ABORTED c=20
 "Print aborted"
-"\x00"
+"Print je prekinut"
 
 #MSG_PREHEATING_TO_LOAD c=20
 "Preheating to load"
-"\x00"
+"Predgrij za punjenje"
 
 #MSG_PREHEATING_TO_UNLOAD c=20
 "Preheating to unload"
-"\x00"
+"Predgrij. za praznj"
 
 #MSG_SELFTEST_PRINT_FAN_SPEED c=18
 "Print fan:"
-"\x00"
+"Ventilator printa:"
```

Comment:
Is it `fena` or `fana` or `fan` or `ventilatora` or `venti` or `Ventilator`?
Please use same word if possible.

Reply:
Corrected ✔️